<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>670</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1026</width>
    <height>670</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1026</width>
    <height>670</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MathBattle для разработчиков</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(13, 86, 166);</string>
  </property>
  <widget class="QLabel" name="labelStatus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>821</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>821</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(1, 173, 188, 25);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="verticalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <property name="horizontalScrollMode">
    <enum>QAbstractItemView::ScrollPerPixel</enum>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>220</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>13</red>
       <green>86</green>
       <blue>166</blue>
      </color>
     </brush>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Название задачи</string>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>13</red>
       <green>86</green>
       <blue>166</blue>
      </color>
     </brush>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Условие</string>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>13</red>
       <green>86</green>
       <blue>166</blue>
      </color>
     </brush>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ответ</string>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>13</red>
       <green>86</green>
       <blue>166</blue>
      </color>
     </brush>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Логин пользователя</string>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>13</red>
       <green>86</green>
       <blue>166</blue>
      </color>
     </brush>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Кол-во репортов</string>
    </property>
    <property name="foreground">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>13</red>
       <green>86</green>
       <blue>166</blue>
      </color>
     </brush>
    </property>
   </column>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>40</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>141</horstretch>
     <verstretch>51</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>181</width>
     <height>71</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>181</width>
     <height>71</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>141</width>
     <height>71</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>141</width>
     <height>51</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 1px solid black;
	color: rgb(255, 255, 255);
	background-color:  rgb(39, 79, 125);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(13, 86, 166);
}

QPushButton:pressed {
	background-color:  rgb(24, 50, 79);
}

QGroupBox {
	color: rgb(255, 255, 255);
	border-radius: 3px;
	border: 1px solid white;
}</string>
   </property>
   <property name="title">
    <string>Поиск задания по ID:</string>
   </property>
   <widget class="QLineEdit" name="lineSearch">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;
background-color: rgb(81, 144, 226);
border-radius: 5px;  </string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonFindTask">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>43</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>161</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>161</width>
      <height>21</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>171</width>
      <height>27</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>171</width>
      <height>27</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="toolTip">
     <string>Найти задачу по ID</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;
border-radius: 5px;  </string>
    </property>
    <property name="text">
     <string>Найти</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ButtonExit">
   <property name="geometry">
    <rect>
     <x>854</x>
     <y>635</y>
     <width>162</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>162</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>162</width>
     <height>25</height>
    </size>
   </property>
   <property name="toolTip">
    <string>Выйти из аккаунта</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
border: 1px solid black;	
border-radius: 5px;
background-color: rgb(255, 154, 0);
gridline-color: rgb(166, 100, 0);
}

QPushButton:hover {
	background-color: rgb(255, 174, 10);
}

QPushButton:pressed {
	background-color: rgb(225, 124, 0);
}</string>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
  <widget class="QLabel" name="Nickname_small">
   <property name="geometry">
    <rect>
     <x>854</x>
     <y>604</y>
     <width>162</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>162</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>162</width>
     <height>25</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black;
color: rgb(255, 255, 255);
background-color: rgb(39, 79, 125);
border-radius: 5px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Nickname</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>120</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	color: rgb(255, 255, 255);
	border-radius: 3px;
	border: 1px solid white;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>162</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QPushButton" name="update_tasks_button">
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 1px solid black;
	color: rgb(255, 255, 255);
	background-color: rgb(81, 144, 226);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(96, 159, 241);
}

QPushButton:pressed {
	background-color:  rgb(13, 86, 166);
}</string>
       </property>
       <property name="text">
        <string>Обновить</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../Антон/Desktop/data/img/update.jpg</normaloff>../../../../../Антон/Desktop/data/img/update.jpg</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="del_user_button">
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>7</pointsize>
         <weight>9</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 75 7pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid black;	
border-radius: 5px;
background-color: rgb(255, 154, 0);
gridline-color: rgb(166, 100, 0);
}

QPushButton:hover {
	background-color: rgb(255, 174, 10);
}

QPushButton:pressed {
	background-color: rgb(225, 124, 0);
}</string>
       </property>
       <property name="text">
        <string>Удалить user'a</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../Антон/Desktop/data/img/del_user.jpg</normaloff>../../../../../Антон/Desktop/data/img/del_user.jpg</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="open_user_tasks_button">
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	font: 8pt &quot;MS Shell Dlg 2&quot;;
	border: 1px solid black;
	color: rgb(255, 255, 255);
	background-color: rgb(81, 144, 226);
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(96, 159, 241);
}

QPushButton:pressed {
	background-color:  rgb(13, 86, 166);
}</string>
       </property>
       <property name="text">
        <string>Задачи user'а</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../Антон/Desktop/data/img/user.jpg</normaloff>../../../../../Антон/Desktop/data/img/user.jpg</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="del_task_button">
       <property name="minimumSize">
        <size>
         <width>77</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>35</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>7</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 7pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid black;	
border-radius: 5px;
background-color: rgb(255, 154, 0);
gridline-color: rgb(166, 100, 0);
}

QPushButton:hover {
	background-color: rgb(255, 174, 10);
}

QPushButton:pressed {
	background-color: rgb(225, 124, 0);
}</string>
       </property>
       <property name="text">
        <string>Удалить задачи</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../../../Антон/Desktop/data/img/delite.png</normaloff>../../../../../Антон/Desktop/data/img/delite.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
